public/app/features/alerting/unified/components/mute-timings/MuteTimingTimeInterval.tsx

```diff
@@ -22,114 +25,122 @@ export const MuteTimingTimeInterval = () => {
-                        (day) => {
-                          const parsedDay = parseInt(day, 10);
-                          return (parsedDay > -31 && parsedDay < 0) || (parsedDay > 0 && parsedDay < 32);
-                        },
-                        'Invalid day'
-                      ),
-                  })}
-                  className={styles.input}
-                  // @ts-ignore react-hook-form doesn't handle nested field arrays well
-                  defaultValue={timeInterval.days_of_month}
-                  placeholder="Example: 1, 14:16, -1"
-                  data-testid="mute-timing-days"
-                />
-              </Field>
-              <Field
-                label="Months"
-                description="The months of the year in either numerical or the full calendar month"
-                invalid={!!errors.time_intervals?.[timeIntervalIndex]?.months}
-                error={errors.time_intervals?.[timeIntervalIndex]?.months?.message}
-              >
-                <Input
-                  {...register(`time_intervals.${timeIntervalIndex}.months`, {
-                    validate: (value) =>
-                      validateArrayField(
-                        value,
-                        (month) => MONTHS.includes(month) || (parseInt(month, 10) < 13 && parseInt(month, 10) > 0),
-                        'Invalid month'
-                      ),
-                  })}
-                  className={styles.input}
-                  placeholder="Example: 1:3, may:august, december"
-                  // @ts-ignore react-hook-form doesn't handle nested field arrays well
-                  defaultValue={timeInterval.months}
-                  data-testid="mute-timing-months"
-                />
-              </Field>
-              <Field
-                label="Years"
-                invalid={!!errors.time_intervals?.[timeIntervalIndex]?.years}
-                error={errors.time_intervals?.[timeIntervalIndex]?.years?.message ?? ''}
-              >
-                <Input
-                  {...register(`time_intervals.${timeIntervalIndex}.years`, {
-                    validate: (value) => validateArrayField(value, (year) => /^\d{4}$/.test(year), 'Invalid year'),
-                  })}
-                  className={styles.input}
-                  placeholder="Example: 2021:2022, 2030"
-                  // @ts-ignore react-hook-form doesn't handle nested field arrays well
-                  defaultValue={timeInterval.years}
-                  data-testid="mute-timing-years"
-                />
-              </Field>
-              <Button
-                type="button"
-                variant="destructive"
-                icon="trash-alt"
-                onClick={() => removeTimeInterval(timeIntervalIndex)}
-              >
-                Remove time interval
-              </Button>
-            </div>
-          );
-        })}
+        <Stack direction="column" gap={2}>
```

Comment:
Another weird diff; this time because I'm wrapping the `intervals` in a `<Stack />` instead of applying bottom margins. There are some important changes in here though so don't skip past this part of the review.